plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -166,4 +169,28 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    public static IcebergColumnHandle pathColumnHandle()
+    {
+        return new IcebergColumnHandle(
+                columIdentity(FILE_PATH),
+                VARCHAR,
+                ImmutableList.of(),
+                VARCHAR,
+                Optional.empty());
+    }
+
+    public static ColumnMetadata pathColumnMetadata()
+    {
+        return ColumnMetadata.builder()
+                .setName(FILE_PATH.columnName())
+                .setType(VARCHAR)
```

Comment:
Please use the type in `FILE_PATH` instead of `VARCHAR`.

Reply:
updated @ebyhr